src/jade/resources/default_cfg/benchmarks_pp/raw/openmc/FNG-HCPB.yaml

```
@@ -0,0 +1,140 @@
+      - [mass, {}]
+      - [scale, {'factor': 2.57042574E+07}]
+      - [replace, {'column': 'Cells', 'values': *depth_h3}]
+
+Tritium activity Pellet:
+  concat_option: no_action
+  84: 
+      - [mass, {}]
+      - [scale, {'factor': 2.57042574E+07}]
+      - [add_column_with_dict, {'ref_column': 'Cells', 'values': *depth_h3d, new_columns: [ 'Distance','Pellet']}]
+      - [replace, {'column': 'Cells', 'values': *depth_h3}]  
+
```

YAMLlint (1.37.1)

[warning] too many blank lines (1 > 0)

(empty-lines)